src/PeriodicOrbits.jl

```diff
@@ -1,10 +1,18 @@
 module PeriodicOrbits
 
+# Use the README as the module docs
+@doc let
+    path = joinpath(dirname(@__DIR__), "README.md")
+    include_dependency(path)
+    read(path, String)
+end PeriodicOrbits
+
```

Comment:
This is actually not necessary, I just learned. Julia does this automatically for you.

Reply:
So you can delete this code block.

Reply:
Deleting results in a build error

```
┌ Error: no docs found for 'PeriodicOrbits' in `@docs` block in src/index.md:1-3
│ ```@docs
│ PeriodicOrbits
│ ```
└ @ Documenter ~/.julia/packages/Documenter/qoyeC/src/utilities/utilities.jl:44
```

Can you link some example package where it works?